riotfile.py

```diff
@@ -156,22 +156,22 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
-        Venv(
-            pys=["3"],
-            name="mypy",
-            command="mypy {cmdargs}",
-            create=True,
-            pkgs={
-                "mypy": latest,
-                "envier": latest,
-                "types-attrs": latest,
-                "types-docutils": latest,
-                "types-protobuf": latest,
-                "types-PyYAML": latest,
-                "types-setuptools": latest,
-                "types-six": latest,
-            },
-        ),
+        # Venv(
+        #     pys=["3"],
+        #     name="mypy",
+        #     command="mypy {cmdargs}",
+        #     create=True,
+        #     pkgs={
+        #         "mypy": latest,
+        #         "envier": latest,
+        #         "types-attrs": latest,
+        #         "types-docutils": latest,
+        #         "types-protobuf": latest,
+        #         "types-PyYAML": latest,
+        #         "types-setuptools": latest,
+        #         "types-six": latest,
+        #     },
+        # ),
```

Comment:
We should revert this change. This PR should pass mypy checks

Reply:
Reverted this change, didn't mean for this to sneak into the PR